.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "subprojects/quickjs"]
+	path = subprojects/quickjs
+	url = https://github.com/frida/quickjs
+[submodule "subprojects/capstone"]
+	path = subprojects/capstone
+	url = https://github.com/frida/capstone
+[submodule "subprojects/tinycc"]
+	path = subprojects/tinycc
+	url = https://github.com/frida/tinycc
```

Comment:
Nit: Could we put Capstone first, then QuickJS, and finally TinyCC? (Following order of appearance in `meson.build`, which indicates how "important" they are.)